cfme/automate/service_dialogs.py

```diff
@@ -139,6 +140,14 @@ def update(self, updates):
         sel.click(form_buttons.save)
         flash.assert_no_errors()
 
+    def add_another_element(self, element_data):
+        navigate_to(self, 'Edit')
+        tree = accordion.tree("Dialog")
+        tree.click_path(self.label, self.tab_label, self.box_label)
+        self.add_element(element_data)
+        sel.click(form_buttons.save)
+        flash.assert_no_errors()
+
     def update_element(self, second_element, element_data):
         navigate_to(self, 'Edit')
         if self.appliance.version > "5.5":
```

Comment:
Rest of the version pick for less that 5.7 are removed in PR 4688 .
Also the old web_ui calls are replaced by widgetastic in that PR .